### DISCUSSION TITLE
When a feed is available again

### DISCUSSION THREAD

**Discussion by campones** [videojs/video.js#8184] [Ideas]
Hello

Recently I have been testing vjs8 (been using the mangui's fork of vjs5 that allowed playing hls with a flash player since 2015 on my website..)

so one of the cool feature I have seen is that for instance, a stream that is not broadcasting, can start again on its own as soon as the feed is made available again (broadcaster started streaming ) 

so the player automatically starts playing without any user interaction, which is pretty cool and remind me of rtmp streaming actually.  but..  problem is videojs starts broadcasting immediately as soon as an .m3u8 is available but usually there is only one segment or 2 at best available in the playlist, and that can create buffering issue because the playlist wasn't fully filled when it started. 

I don't know if there is any work around for this.  a simple delay added could fix this, like wait 20s to resume the play when a feed is available again.

**Comment by gkatsev** [comment]
Unfortunately, Video.js doesn't have anything for this. I would recommend having the backend not expose the m3u8 until 3 segments are available.

**Reply by campones** [reply]
I am using a free version of nginx-rtmp that is not really being maintained as far as I know , I doubt this is possible

**Reply by campones** [reply]
Or maybe the player could wait to have 3 4 segments in the playlist before starting?

**Comment by campones** [comment]
but yea, that's a great suggestion
